docs/content/docs/deep/componets.md

```diff
@@ -0,0 +1,113 @@
+# Components
+
+Fluid Framework powered applications have several ways to modularize their code and data into separate components.
+
+## Containers
+
+We expect most Fluid applications to to operate on a single container at a time, ans store all their persisted content in that container.
+It is however possible to use multiple containers, and some use-cases may require it.
+Specifically data which require different permissions, or to be stored in different locations, has to be split into separate containers.
+
+When splitting documents/data across multiple containers, it is up to the application to decide when to delete containers which are no longer needed.
+
+Each container is independent synchronization wise and there is no way to do anything atomically or even strongly ordered across multiple containers.
+
+Each container gets its own service client and thus separate op stream, storage and permissions.
+
+If desired, an application can open a second copy of a container as a mechanism to have a separate "branch" (independent view which can have edits speculatively applied).
```

Comment:
This feels like not related to the topic (this and next 2 paragraphs), I'd remove it. For multiple containers to feel like branches, I think one would expect to control outbound queue (i.e. pause it), which is not supported.